### PR TITLE
MAGN-7063 Can't undo group title change

### PR DESCRIPTION
**Pull request by ramramps**
**Issue**
   Group Title requires two undo to get the previous text

**Fix**
   UpdateValueCommand is moved from GroupTextblock to GroupTextbox.

**Test**
   UndoAnnotationText - Added to core test.

- [x] @pboyer

### CODE REVIEW

File: test/DynamoCoreTests/CoreTests.cs [hunk 1]
```diff
@@ -83,6 +83,45 @@ public void CanAddAnnotation()
...
+                        "This is a unit test"));
+            Assert.AreEqual("This is a unit test", annotation.AnnotationText);
+
+            //Undo Annotation text
+            ViewModel.UndoCommand.Execute(null);
+
+            //Title should be changed now.
+            Assert.AreEqual("This is a test group", annotation.AnnotationText);
+        }
+
```
Comment (pboyer) [inline test/DynamoCoreTests/CoreTests.cs]: Is it possible for this test to inherit from `DynamoModelUnitTestBase`?  Then we don't need to construct a `ViewModel` layer.  Beyond removing a bunch of code-noise (e.g. `Model` as opposed to `ViewModel.Model`) this means your test can actually be executed on Linux or Mac.

**Comment by pboyer** [comment]
This looks good, but it would be much better to use just the model layer to do this testing.  This may be a tough change, but this will need to be done eventually.

**Comment by ramramps** [comment]
@pboyer - Added a new test file for Groups - this inherits from a model. If this is OK, then I will change all the Group Unit tests to inherit from model (that is I will have the tests only in this file).

**Comment by pboyer** [comment]
I've merged this to aid in testing.

**Pull request merged**